sunpy/map/compositemap.py

```diff
@@ -2,18 +2,22 @@
 
 Author: `Keith Hughitt <keith.hughitt@nasa.gov>`
 """
+import matplotlib
 import matplotlib.pyplot as plt
 from matplotlib.collections import Collection, QuadMesh
 from matplotlib.contour import ContourSet, QuadContourSet
 from matplotlib.image import AxesImage, _ImageBase
 
 import astropy.units as u
+from astropy.utils.introspection import minversion
 
 from sunpy.map import GenericMap
 from sunpy.util import expand_list, get_keywords, get_set_methods
 from sunpy.util.decorators import add_common_docstring
 from sunpy.visualization import axis_labels_from_ctype, peek_show, wcsaxes_compat
 
+MATPLOTLIB_LT_3_8 = not minversion(matplotlib, "3.8.dev")
```

Comment:
Stolen with love from Astropy https://github.com/astropy/astropy/pull/15004/files

Reply:
Remove the `not` and invert your conditional below?

Reply:
Sure.